src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -73,4 +73,8 @@ public static Tag parseTag(String tag) throws ParseException {
         }
         return tagSet;
     }
+
+    public static Integer parseCount(String s) {
```

Comment:
I think we should account for non-integer strings. Maybe the function can throw a `ParseException` if `s` cannot be parsed into a positive integer?